7_react/react1/week1/src/todo_list/TodoList.jsx

```diff
@@ -0,0 +1,51 @@
+import { TodoItem } from "../todoItem/TodoItem";
+const AppTodo = () => {
+    
+    const todos = [
+      {
+        name: "Get out of bed",
+        deadline: "Wed Sep 13 2017",
+        status: true,
+      },
+      {
+        name: "Brush teeth",
+        deadline: "Thu Sep 14 2017",
+        status: false,
+      },
+      {
+        name: "Eat breakfast",
+        deadline: "Fri Sep 15 2017",
+        status: false,
+      },
+    ];
+
+   
+
+    const TodoList = (props) => {
+        console.log(props);
+
+        return (
+
+            <ul>
+                {props.todos.map((todo) => {
+                    return (
+                        <TodoItem
+                            name={todo.name}
+                            deadline={todo.deadline}
+                            status={todo.status}
+                        />
+                    )
+                })};
```

Comment:
you can delete this `;` :)
<img width="524" alt="Screenshot 2024-03-18 at 16 34 02" src="https://github.com/HackYourFuture-CPH/class27-homework/assets/48878547/12fcc80b-28ff-4ce8-bc09-7b2f50953f02">